components/automate-cli/pkg/verifyserver/constants/certificatevalidation.go

```diff
@@ -2,10 +2,11 @@ package constants
 
 const (
 	CERTIFICATE_EXPIRY_TITLE                 = "Certificates have valid expiry date"
-	CERTIFICATE_EXPIRY_SUCCESS_MESSAGE       = "All certificates expiry date is later than 365 days"
+	CERTIFICATE_VALID                        = "Certificates are valid"
+	CERTIFICATE_EXPIRY_SUCCESS_MESSAGE       = "All certificates expiry date is later than 30 days"
```

Comment:
If we are not using CERTIFICATE_EXPIRY_SUCCESS_MESSAGE anywhere, then we can remove this

Reply:
@Sahiba3108  we are using this